jdi-reportportal-example/src/test/resources/reportportal.properties

```diff
@@ -0,0 +1,8 @@
+rp.enable = true
```

Comment:
where should we put reportportal url etc?

Reply:
added properties for rp credentials in latest commit